packages/hub/src/scope.ts

```diff
@@ -224,9 +224,6 @@ export class Scope implements ScopeInterface {
    */
   public static clone(scope?: Scope): Scope {
     const newScope = new Scope();
-    Object.assign(newScope, scope, {
```

Comment:
I have no clue why we had it here if we manually assign all the properties below anyway.
Same goes for `_scopeListeners`, which is initialized in the constructor and we don't have to override it if we skip `Object.assign`.